index.js

```diff
@@ -0,0 +1,2 @@
+require('babel-register');
```

Comment:
I seem not to be comfortable with you still having 2 lines of requiring modules here. Why haven't you changed this 'require' to ES6